esp/scm/ws_workunits.ecm

```diff
@@ -2256,7 +2256,7 @@ ESPservice [
     ESPmethod [cache_seconds(60), min_ver("1.61")] WUGetNumFileToCopy(WUGetNumFileToCopyRequest, WUGetNumFileToCopyResponse);
     ESPmethod [min_ver("1.71")] WUDetails(WUDetailsRequest, WUDetailsResponse);
     ESPmethod [min_ver("1.71")] WUDetailsMeta(WUDetailsMetaRequest, WUDetailsMetaResponse);
```

Comment:
This should possibly be cached with a high timeout - it should never change while the system is up.